Packs/SumoLogic_Cloud_SIEM/pack_metadata.json

```diff
@@ -0,0 +1,20 @@
+{
+    "name": "Sumo Logic Cloud SIEM",
+    "description": "Cloud SIEM with SOC analytics and automation.",
```

Comment:
-> " Sumo Logic Cloud SIEM provides threat detection and incident response for modern IT environments. This content pack will allow you to apply automation to perform actual SOC analyst workflows.  Using this content pack you will be able to fetch Incidents via Insights, update status of an Insight, add items to match list, add Threat Intel Indicators to Threat Intel Sources, and so on."